devices/models.py

```diff
@@ -100,6 +102,35 @@ class Bookmark(models.Model):
     user = models.ForeignKey(Lageruser)
 
 
+class Vendor(models.Model):
+    name = models.CharField(_('Name'), max_length=100)
+    driver_url = models.CharField(_('Drivers'), max_length=1000, blank=True, null=True)
+    support_url = models.CharField(_('Support'), max_length=1000, blank=True, null=True)
+
+    class Meta:
+        permissions = (
+            ("access_vendor", _("Can access Vendor")),
+        )
+
+    def __str__(self):
+        return self.name
+
+    def get_absolute_url(self):
+        return reverse('vendor-detail', kwargs={'pk': self.pk})
+
+    def get_edit_url(self):
+        return reverse('vendor-edit', kwargs={'pk': self.pk})
+
+    def get_associated_devices(self):
+        return Device.objects.filter(vendor=self)
+
+    def clean(self):
+        if "SERVICETAG" not in self.driver_url:
```

Comment:
The name "SERVICETAG" is a specific name for a device property of a specific manufacturer/vendor (Dell).
Ideally this needs to be dynamically added to each URL. Some vendor links may require the serial number, some the exact product name and so on.

Iff you decided to hard code these properties, I think they should at least be given the generic name (here: serialnumber).